### ISSUE TITLE
Outreachy Documentation Project: Anita Igbine

### ISSUE THREAD

**Issue by AIgbine** [ersilia-os/ersilia#155]
Applicant: @AIgbine
  
Welcome to the Ersilia Open Source Initiative. This issue will serve to track all your contributions for the project “Improve the documentation and outreach material of the Ersilia Model Hub”.
  
Please tick the tasks as you complete them. To make a final application it is *not* required to have completed all tasks. Only the *Initial Steps* and *Community* sections are REQUIRED. The tasks are not ordered from more to less important, they are simply related to different skills. Start where you feel most comfortable. This project can be adapted to the applicants interests, please focus on the type of tasks that you prefer / have better skills / would like to work on as an intern.

---
### Initial steps:
- [x] Record your application for the project in the outreachy website referencing this issue. Please make sure to select the right project on the website.
- [x] Join the Slack channel to follow public communications
- [x] Comment under this issue explaining why are you interested in this project
---
### GitHub documentation:
- [ ] Create a README file with the name <README_yourname.md> under the /documentation folder
- [ ] Link the #PR in a comment under this issue
- [ ] Incorporate feedback from the mentor
---
### Writing dissemination material
- [x] Read the [Strategic plan 2021-2023](https://ersilia.gitbook.io/ersilia-wiki/) for Ersilia and create a 1-page blogpost with the main points
- [x] Comment under this issue with a link to the blogpost (a google docs for example)
- [x] Incorporate feedback from the mentor
- [x] Choose your own topic related to Ersilia (AI/ML for biomedical research, neglected diseases, drug discovery…) and write a 1-page blogpost to communicate to a non-expert audience
- [x] Comment under this issue with a link to the blogpost (a google docs for example)
- [x] Incorporate feedback from the mentor
- [x] Create a template for a twitter post to release every time a new model is incorporated in the Hub (twitter: 280 characters, you can suggest a main post +  thread with extra information) and add it as a comment under this issue
- [x] Create a template short Newsletter (1 paragraph) to send every month to our community (funders, users, contributors). It should mention metrics (models in the hub, number of users, funding…), thank you etc 
---
### Technical skills (required for the tutorial only)
- [x] [Install](https://ersilia.gitbook.io/ersilia-book/quick-start/installation) the Ersilia Model Hub
- [ ] Test one model
- [ ] Add a screenshot under this issue showing the model running in your computer
- [ ] Write a docstring for the ErsiliaModel class. Use the Google Python Style guide. Paste the docstring as a comment below (do not use a PR).
---
### Graphic material
- [x] Read the [Ersilia Brand Guidelines](https://medium.com/ersiliaio/ersilia-open-source-initiative-brand-guidelines-130fe53104f5)
- [x] Read [“Why Ersilia?”](https://ersilia.gitbook.io/ersilia-wiki/)
- [x] Create one image / slide to explain Ersilia’s mission and vision
- [x] Link to the image/slide as a comment under this issue
- [x] Incorporate feedback from the mentor
- [ ] Create two slides / short video showing how to use the Ersilia Model Hub and add them under the /tutorial folder 
- [ ] Link the #PR in this issue
- [ ] Incorporate feedback from the mentor
---
### Scientific content
- [x] Check the models available in the [Hub](https://airtable.com/shrXfZ8pqro0jjcsG/tblZGe2a2XeBxrEHP/viwd5XJVLslkE11Tg)
- [x] Select one model from the list and write a technical card (what is the model for, what input, which data was used to create it, what kind of ML algorithm uses…) for it
- [x] Add your card as a comment to this issue
- [ ] Search the scientific literature and suggest 3 new models (comment in this issue) that would be relevant to incorporate in the Hub.
---
### Community
- [ ] Look up two other projects and comment on their issues with feedback on one of their tasks
- [x] If you have feedback from your peers, answer it in this issue.
---
### Other
If you have interest in working on related topics, or have new suggestions, please do the following
- [ ] Add a comment in this issue with your new idea, tagging the mentor 
- [ ] Get feedback from the mentor and act accordingly
- [ ] Link in the comments any other PR you have contributed to.
---
### Final application
- [x] I have answered all comments from mentors and contributors
- [x] All PR or issues assigned to me are complete
- [x] I have submitted my final application to the project

**Comment by AIgbine** [comment]
Hi, I'm Anita. A data analyst and technical writer. I am contributing to this documentation for the following reasons:

1. I started out learning python on my own and documentation, how-to tutorials were my go-to whenever I got stuck
2. I enjoy contributing to learning resources and have done so for a year now
3. For my final year project at school (class of 2020) I build a machine learning model. It gave me an opportunity to learn the basics of building a model and intend to use this knowledge to contribute to this project.
4. I would love to contribute to Open Source. It's something that scares and excites me.

**Comment by GemmaTuron** [comment]
Hi @AIgbine !

Thanks for your interest! Given your background and description, perhaps your skills are better suited to the coding project? If you have python knowledge and are interested in the computational aspects of our work check the second open project!

**Comment by AIgbine** [comment]
Hi @GemmaTuron

I will definitely be contributing to the coding project. On the outreachy website I noticed that you only need one intern so I plan to contribute to both the documentation and the coding projects. 

I'll jump on the second project right away!

**Comment by AIgbine** [comment]
Hi @GemmaTuron 

Here is my link for a one-page blog on the Strategic Plan 2021 - 2023

https://docs.google.com/document/d/1lM3w3Ojdmj3edkneJi4XjjPzfNrk9pjJtNl1aUndC20/edit

**Comment by victorabba** [comment]
Hi @AIgbine. Your Blogpost is superb, but you need to add a title to the blog.

**Comment by AIgbine** [comment]
Hey @victorabba, 

I put the title as the document name initially but I have now put it at the top of the post
Thank you for taking time out to read it!

**Comment by GemmaTuron** [comment]
Hi @Algbine!

Good job on this post! I left some comments for your consideration, check them before going onto other tasks but continue this good work, thanks!

**Comment by AIgbine** [comment]
Hey @GemmaTuron 

I've implemented the corrections. Thank you for taking time to read through the document
https://docs.google.com/document/d/1lM3w3Ojdmj3edkneJi4XjjPzfNrk9pjJtNl1aUndC20/edit

**Comment by AIgbine** [comment]
Hi @GemmaTuron 

Here is my submission for the Ersilia newsletter template: https://docs.google.com/document/d/1V27kRIyaNNsRuW4N3hBVX3ttctjvZKxFuSuw7bTyyyY/edit?usp=sharing

I have also written a one-page blog post on Drug Discovery here: https://docs.google.com/document/d/11Pig-9SFFV5LZQeTJsHhsZ2X7EEuGfPyMPx5KeDR4cA/edit?usp=sharing

**Comment by AIgbine** [comment]
@GemmaTuron, had to put this in a separate comment:

Here is a template for announcing new models on Twitter:

New at Ersilia!

We’re happy to introduce the drug-discovery-ai model. Credit to @xxis and her team for working hard on this model. The drug-discovery-ai model improves drug discovery with an accuracy of more than 98%. Head over to the Ersilia Model Hub to use this model! [Link] (1/2)

Not sure where to start? The model documentation will help you get your model running in no time! Read the model documentation here or watch this video on Youtube to get started. [Link to resources] (2/2)


[The main point is to introduce the model in a catchy way while briefly stating its benefits and lastly, providing a link to the model. Further information can be given in subsequent threads or a simple link to the documentation and other resource materials as I have done here]

**Comment by AIgbine** [comment]
> @AIgbine, thanks for the Tag (Xxis - old account), but I is not Her. and would be a shame for them to miss out on the praise tag!

Oh my. I actually just came up with it as a place holder. Never thought it would belong to anyone 😅

**Comment by GemmaTuron** [comment]
Hi @AIgbine 

Thanks for the effort in the project! Well done in the blogpost, and I like that you introduce new funders and donors for the newsletter. 
Can I ask you to add here the Mission and Vision template as a .png or .jpg file for easy download?
You have done a lot of work. To complete your application, I would suggest to focus in one of the tasks of the Scientific content, and you will be ready for making the final application on the website.
Thanks for the effort

**Comment by AIgbine** [comment]
Thanks for going through my work @GemmaTuron!

Here is the .png file of the Missio
![Ersilia Mission and Vision_ AIgbine](https://user-images.githubusercontent.com/35337868/163131688-463ea8b7-8983-4563-8483-3b29896f8c11.png)
n and Vision

**Comment by AIgbine** [comment]
@GemmaTuron here is my submission for model technical card

**Model Summary**
**Name**: chemprop-antibiotic
**Code**: e0s4e40
**Function**: The model predicts the antibiotic activity of molecules that are structurally different from known antibiotics. 
**Training Data**: The model training data consists of data available from US Food and Drug Administration (FDA) data and natural products isolated from plants, animals, and microbial sources. A total of 2335 unique compounds.
**Algorithm**: Deep neural network model

**Input and Output**
Input: A compound
Output: Antibiotic activity of the compound

**Resources**
Publication: https://pubmed.ncbi.nlm.nih.gov/32084340/
GitHub: https://github.com/ersilia-os/eos4e40
Source: http://chemprop.csail.mit.edu/checkpoints

**License**
MIT

**Comment by AIgbine** [comment]
@GemmaTuron  here is my submission for model technical card

**Model Summary**
Name: chemprop-antibiotic
Code: e0s4e40
Input: A compound
Output: Antibiotic activity of the compound
Function: The model predicts the antibiotic activity of molecules that are structurally different from known antibiotics. 

**Data and Algorithm**
Training Data: The model training data consists of data available from US Food and Drug Administration (FDA) data and natural products isolated from plants, animals, and microbial sources. A total of 2335 unique compounds.
Algorithm: Deep neural network model

**Resources**
Publication: https://pubmed.ncbi.nlm.nih.gov/32084340/
GitHub: https://github.com/ersilia-os/eos4e40
Source: http://chemprop.csail.mit.edu/checkpoints

**License**
MIT

**Comment by pauline-banye** [comment]
> @GemmaTuron, had to put this in a separate comment
> 
> Here is a template for announcing new models on Twitter:
> 
> New at Ersilia!
> 
> We’re happy to introduce the drug-discovery-ai model. Credit to @[account name/just name] and her team for working hard on this model. The drug-discovery-ai model improves drug discovery with an accuracy of more than 98%. Head over to the Ersilia Model Hub to use this model! [Link] (1/2)
> 
> Not sure where to start? The model documentation will help you get your model running in no time! Read the model documentation here or watch this video on Youtube to get started. [Link to resources] (2/2)
> 
> [I introduced the model in a catchy way while briefly stating its benefits and lastly, providing a link to the model. Further information can be given in subsequent threads or a simple link to the documentation and other resource materials as I have done here]

Well done @AIgbine with the twitter template. I loved that you gave credit where due. I did feel that a link to a YouTube video would be perfect here.

**Comment by AIgbine** [comment]
> > @GemmaTuron, had to put this in a separate comment
> > Here is a template for announcing new models on Twitter:
> > New at Ersilia!
> > We’re happy to introduce the drug-discovery-ai model. Credit to @[account name/just name] and her team for working hard on this model. The drug-discovery-ai model improves drug discovery with an accuracy of more than 98%. Head over to the Ersilia Model Hub to use this model! [Link] (1/2)
> > Not sure where to start? The model documentation will help you get your model running in no time! Read the model documentation here or watch this video on Youtube to get started. [Link to resources] (2/2)
> > [I introduced the model in a catchy way while briefly stating its benefits and lastly, providing a link to the model. Further information can be given in subsequent threads or a simple link to the documentation and other resource materials as I have done here]
> 
> Well done @AIgbine with the twitter template. I loved that you gave credit where due. I did feel that a link to a YouTube video would be perfect here.

Thanks for the review @pauline-banye. Yes, that's a good idea. YouTube videos and other resources can be linked to the second tweet.

**Comment by GemmaTuron** [comment]
Hi @AIgbine 

Thanks for the technical card, really well-thought!
Now focus on creating a good final application in the Outreachy website please!
